test/download-file.spec.js

```diff
@@ -0,0 +1,56 @@
+import expect from 'expect';
+import sinon from 'sinon';
+import fs from 'fs';
+
+import { writeImageOnDisk } from '../src/download-file';
+
+/* eslint-disable  func-names, prefer-arrow-callback */
+describe('writeImageOnDisk', function() {
```

Comment:
Sure. So basically we don't want to lose context when we do the tests. You can take a look at: [Arrow functions in mocha](http://mochajs.org/#arrow-functions)